libs/partners/nvidia-trt/tests/integration_tests/test_llms.py

```diff
@@ -8,7 +8,7 @@
 
 @pytest.mark.skip(reason="Need a working Triton server")
 def test_stream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from nvidia-trt."""
```

Comment:
and they go all uppercase
```suggestion
    """Test streaming tokens from NVIDIA TRT."""
```